draft-ietf-detnet-dp-sol-mpls-00.xml

```diff
@@ -181,7 +192,7 @@
 
     <t hangText="PROF">
      A Packet Reorder Function (PROF) re-orders packets within a DetNet flow  that are received out of order.
-     This function may be implemented at an edge or a relay node.
+     This function may be implemented at an edge node or end system.

```

Comment:
Is there a  specific reason why the relay node cannot do this function anymore?

Reply:
My understanding of the architecture definitions are that DetNet end
systems and edge nodes can provide DetNet service layer functions while
relay on provide DetNet transport layer functions -- and this is the
core difference between edge and relay nodes. Am I wrong?